### PR TITLE
server bugs fixed

### PR DESCRIPTION
**Pull request by abhishek9celestial**

### CODE REVIEW

File: cypress/e2e/IPSTACK/blog.cy.js [hunk 1]
```diff
@@ -36,8 +36,13 @@ describe('Ipstack Blog page', () => {
...
     });
 
-    it('8. Test to click on the footer link of "Blog"', () => {
-        cy.get('.elementor-element-c2f551c > .elementor-widget-container > .elementor-icon-list-items > :nth-child(2) > a > .elementor-icon-list-text').click();
+    it('8. Test to go back to "IPSTACK" main page', () => {
+        cy.navigateUrlwithCookies(IPSTACK_BASE_URL);
+    });
+
+    it('9. Test to click on the footer link of "Blog"', () => {
+        cy.get('.custom-container > :nth-child(3) > :nth-child(3) > a').scrollIntoView();
```
Comment (Karthikkdl) [inline cypress/e2e/IPSTACK/blog.cy.js]:
```suggestion
        const linkSelector = '.custom-container > :nth-child(3) > :nth-child(3) > a';

cy.get(linkSelector).scrollIntoView();
cy.get(linkSelector).click();
```

File: cypress/e2e/IPSTACK/blog.cy.js [hunk 2]
```diff
@@ -36,8 +36,13 @@ describe('Ipstack Blog page', () => {
...
+        cy.get('.custom-container > :nth-child(3) > :nth-child(3) > a').click();
```
Comment (Karthikkdl) [inline cypress/e2e/IPSTACK/blog.cy.js]: you can remove if you think above suggestion works

File: cypress/e2e/zenscrape/login.cy.js [hunk 3]
```diff
@@ -7,18 +7,17 @@ describe('Zenscrape Login page  ', () => {
...
+    cy.componentVisiblityCheck('.card-header','Login');
   });
-
+  
   it('2. Test to check the "Login" page should match the actual URL, if not redirect to the actual URL to ensure other test cases do not fail', () => {
     cy.url().then((currentUrl) => {
       if (!currentUrl.includes('/login')) {
         // Redirect to the actual URL if not already there
-        cy.visit('/login');
+        cy.visit('https://app.zenscrape.com/login');
```
Comment (Karthikkdl) [inline cypress/e2e/zenscrape/login.cy.js]: Why are we hardcoding the URL here?

File: cypress/e2e/zenscrape/status.cy.js [hunk 4]
```diff
@@ -136,6 +136,6 @@ describe('Zenscrape Status page', () => {
     });
 
     it('31. Test to check "Timezone" dropdown is present in the footer', () => {
-        cy.componentVisiblityCheck('.tz__input-container');
+        cy.componentVisiblityCheck('.tz__value-container');
```
Comment (Karthikkdl) [inline cypress/e2e/zenscrape/status.cy.js:139]: what change did you make here? @abhishek9celestial

**Review by Karthikkdl — CHANGES_REQUESTED** [review]

File: cypress/e2e/zenscrape/login.cy.js (hunk 3 above)
Comment (nidhiya-pious) [reply cypress/e2e/zenscrape/login.cy.js]: The base url((https://zenscrape.com/login)) is different and when we click on login button, the url is changing to https://app.zenscrape.com/login.

File: cypress/e2e/zenscrape/status.cy.js (hunk 4 above)
Comment (nidhiya-pious) [reply cypress/e2e/zenscrape/status.cy.js:139]: Selector got changed and that's why it is failing. Now @abhishek9celestial changed it.

File: cypress/e2e/zenscrape/login.cy.js (hunk 3 above)
Comment (NaveenK15) [reply cypress/e2e/zenscrape/login.cy.js]: @nidhiya-pious add this is the data and use a variable here.

**Pull request merged**